crates/turborepo-lib/src/commands/daemon.rs

```diff
@@ -193,6 +196,26 @@ async fn clean(
     }
 }
 
+async fn clean_logs(log_folder: &AbsoluteSystemPath) -> Result<(), DaemonError> {
+    let mut success = true;
+    trace!("cleaning up log files");
+    // clear all files in the log folder. we want to keep the
+    // folder just remove the contents
+    // `remove_dir_all_recursive` is lifted from `std`
+    if let Err(e) = fs::remove_dir_all(log_folder.as_std_path()) {
```

Comment:
I think we should generally prefer using the FS ops through our path lib as we use `fs_err` which includes the path name in the error.
```suggestion
    if let Err(e) = log_folder.remove_dir_all() {
```